### PR TITLE
doc: update dependencies

### LINKED ISSUE #915: [MAINT] Fix installation page
FYI: https://mne.tools/mne-bids/dev/install.html still shows diff dependencies then what's in the setup.cfg.

PS: Also, I'm down to move away from python 3.7 at some point that MNE does: https://github.com/mne-tools/mne-python

### PR DESCRIPTION
**Pull request by sappelhoff**
closes #915 

we will then need to push a commit to maint/0.9 and update the stable docs as well (will do once this is approved and merged)

Merge checklist
---------------

Maintainer, please confirm the following before merging:

- [ ] All comments resolved
- [ ] This is not your own PR
- [ ] All CIs are happy
- [ ] PR title starts with [MRG]
- [ ] [whats_new.rst](https://github.com/mne-tools/mne-bids/blob/main/doc/whats_new.rst) is updated
- [ ] PR description includes phrase "closes <#issue-number>"

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -43,11 +43,10 @@ include_package_data = True
 
 [options.extras_require]
 full =
-  nibabel >= 2.2
+  nibabel >= 2.5
   pybv >= 0.6
   matplotlib >= 3.1.0
   pandas >= 0.24.0
-  openneuro-py >= 2021.8
```
Comment (sappelhoff) [inline setup.cfg:50]: I remove openneuro-py here because we only need it for the documentation (like nilearn, etc.)

Comment (agramfort) [reply setup.cfg:50]: well full means also doc to me. It's also pure Python package. Otherwise we can do a [doc] install target

Comment (sappelhoff) [reply setup.cfg:50]: okay, but then we'd need a couple of more packages there as well right? sphinx, nilearn, sphinx-copybutton, ... https://github.com/mne-tools/mne-bids/blob/ff5cdeda3772bb8e7a82fcd1caf4a5584edc40ec/doc/requirements.txt#L1-L10

Comment (hoechenberger) [reply setup.cfg:50]: To me, full means full functionality for a user, but no dev deps. Building docs is a dev operation. I don't think full needs to support this

**Review by hoechenberger — APPROVED** [review]

File: setup.cfg (hunk 1 above)
Comment (sappelhoff) [reply setup.cfg:50]: I rather agree with @hoechenberger here - but Richard: why would you then include openneuro-py in "full"?

**Review by sappelhoff — COMMENTED** [review]
related: https://github.com/conda-forge/mne-bids-feedstock/pull/18 and https://github.com/conda-forge/staged-recipes/pull/17064

**Comment by hoechenberger** [comment]
Thanks @sappelhoff!

**Comment by sappelhoff** [comment]
Done in 
- https://github.com/mne-tools/mne-bids/commit/79a4d05411cb2b91d3b7d2c28b7b02d05a6c82cd
- https://github.com/mne-tools/mne-bids/commit/b4250ec348a80ac14ec8a14f1a51b0a30afdec8b

once the conda feedstock is updated this should all be fine again

**Pull request merged**